core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -66,6 +66,15 @@ public static long propertyAsLong(
     return defaultValue;
   }
 
+  public static Long propertyAsLong(
```

Comment:
looks like the only use case is when the default is null? I think we can name this something as `propertyAsNullableLong` and remove the default value parameter?

Reply:
Thank you for your suggestions. I've updated the util method.